lib/ui/pages/workspace_page/components/drawing_surface/drawing_canvas.dart

```diff
@@ -26,6 +24,7 @@ class _DrawingCanvasState extends ConsumerState<DrawingCanvas> {
   var _pointersOnScreen = 0;
   var _isZooming = false;
   Offset _lastPointerUpPosition = Offset.zero;
+  final textController = TextEditingController();
```

Comment:
I feel like this is misplaced tool specific controller should be in a tool widget. Its not used anyway.